extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -204,7 +224,31 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					}
 					if (connectionString) {
 						try {
-							const success = await azureFunctionsUtils.setLocalAppSetting(path.dirname(projectUri.fsPath), newConnectionStringSettingName, connectionString);
+							const projectFolder: string = path.dirname(projectUri.fsPath);
+							const localSettingsPath: string = path.join(projectFolder, constants.azureFunctionLocalSettingsFileName);
+							// Connection string does not include a password, so will prompt user to optionally enter it manually
+							if (!connectionInfo?.password || includePassword === constants.noString && connectionInfo.authenticationType !== 'SqlLogin') {
```

Comment:
Shouldn't this only be for authType === 'SqlLogin'? There's no password to enter if it's not basic auth